code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -117,6 +121,7 @@
 	item_state = "chickensuit"
 	body_parts_covered = CHEST|ARMS|GROIN|LEGS|FEET
 	flags_inv = HIDESHOES|HIDEJUMPSUIT
+	identity_name = "chicken?"

```

Comment:
Not strictly this but I have put question marks in names using the identity manager and it worked then, i can always test this